qa/L0_model_namespacing/test_duplication/addsub_repo/composing_model/1/model.py

```diff
@@ -3,4 +3,3 @@
 
 # load pre-defined QA model
 sys.path.append(os.environ["TRITON_QA_PYTHON_MODEL_DIR"])
-from python_addsub import *
```

Comment:
I think this line should be kept here and everywhere. cc @GuanLuo

Reply:
Good catch! Removed.